modules/apps/portal-cache/portal-cache-test/src/testIntegration/java/com/liferay/portal/cache/internal/test/PortalCacheExtenderTest.java

```diff
@@ -63,9 +65,55 @@ public void tearDown() throws Exception {
 		}
 	}
 
+	@Test
+	public void testFinderCache() throws Exception {
+		_multiVmXML = _generateXMLContent(
+			1,
```

Comment:
@marianoalvarosaiz, please rename the test method name for `testFinderCacheConfigAfterUpdateEntityCacheConfig`. Or you may rename one more specific/better than this.